parameters.yml

```diff
@@ -0,0 +1,21 @@
+qm:
```

Comment:
This should likely be moved into the `quest` folder. Maybe title it `default_parameters.yml`.